python/lsst/sims/maf/driver/runBundle.py

```diff
@@ -0,0 +1,96 @@
+import os, re, copy, warnings
+import lsst.sims.maf.sliceMetrics as sliceMetrics
+import lsst.sims.maf.db as db
+import lsst.sims.maf.utils as utils
+
+def runBundle(mafBundle, verbose=True, makePlots=True):
+    """
+    mafBundle should be a dict with the following key:value pairs:
+    'metricList':List of MAF metric objects
+    'slicer': instatiated MAF slicer
+    'dbAddress': string that gives the address to the desired DB
```

Comment:
Also -- if 'runBundle' needs these things, why not just make them args/kwargs to runBundle itself, rather than forcing the user to make a dictionary and then pass a dictionary? Is there any other use for the 'mafBundle' dictionary?